### DISCUSSION TITLE
Can I remove that bar,I am using i3wm with arch linux edit:sorry for the question I should have checked the stuff before asking

### DISCUSSION THREAD

**Discussion by DrNyaaa** [xgi/houdoku#185] [Other]
![image](https://user-images.githubusercontent.com/99406142/185764394-4d9fa86b-853d-494a-8b63-2cac3f1db442.png)

**Comment by yash-278** [comment]
You can fullscreen the app to hide the title bar & You can press `m` to hide the menu bar.
